lib/sentry/event.ex

```diff
@@ -52,15 +54,33 @@ defmodule Sentry.Event do
       |> String.trim("*")
       |> String.trim
 
+    release = case Application.fetch_env(:sentry, :release) do
+      {:ok, r} -> r
+      :error -> nil
+    end
+
+    server_name = case Application.fetch_env(:sentry, :server_name) do
+      {:ok, r} -> r
+      :error -> nil
+    end
+
+    env = case Application.fetch_env(:sentry, :environment_name) do
```

Comment:
```elixir
env = Application.get_env(:sentry, :environment_name, String.to_atom System.get_env("MIX_ENV"))
```

I think we would also have to convert to atom, as environment variables will always be strings.